plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/java/org/eclipse/che/plugin/java/plain/server/generator/PlainJavaProjectGenerator.java

```diff
@@ -71,10 +94,57 @@ public void onCreateProject(
 
     String mainJavaWsPath = resolve(sourceDirWsPath, FILE_NAME);
     fsManager.createFile(
-        mainJavaWsPath,
-        getClass().getClassLoader().getResourceAsStream("files/main_class_content"));
+        mainJavaWsPath, getClass().getClassLoader().getResourceAsStream(MAIN_CLASS_RESOURCE));
+
+    createClasspath(projectWsPath, sourceFolders.get(0));
+    createProjectConfig(projectWsPath);
+  }
+
+  private void createProjectConfig(String projectWsPath)
+      throws ConflictException, NotFoundException, ServerException {
+    InputStream projectIS = getClass().getClassLoader().getResourceAsStream(PROJECT_FILE_RESOURCE);
```

Comment:
It is better to use try with resources for streams

Reply:
fixed